channels/serializers.py

```diff
@@ -195,22 +195,22 @@ class DepartmentChannelSerializer(FieldChannelBaseSerializer):
     department_detail = ChannelDepartmentDetailSerializer()
 
 
-class ChannelOfferorDetailSerializer(serializers.ModelSerializer):
+class ChannelUnitDetailSerializer(serializers.ModelSerializer):
     """Serializer for the ChannelOfferorDetail model"""
 
-    offeror = LearningResourceOfferorDetailSerializer(read_only=True)
+    unit = LearningResourceOfferorDetailSerializer(read_only=True)
 
     class Meta:
-        model = ChannelOfferorDetail
+        model = ChannelUnitDetail
         exclude = ("channel", *COMMON_IGNORED_FIELDS)
 
 
-class OfferorChannelSerializer(FieldChannelBaseSerializer):
+class UnitChannelSerializer(FieldChannelBaseSerializer):
     """Serializer for Channel model of type offeror"""
```

Comment:
Noticed this in the OpenAPI spec
```suggestion
    """Serializer for Channel model of type unit"""
```